docs/getting-started.adoc

```diff
@@ -5,7 +5,7 @@ This document walks you through the development and deployment of a Node.js appl
 == Developing and deploying an application to an OpenShift cluster
 To develop and deploy a Node.js application to an OpenShift cluster using `odo`:
 
-. link:/docs/installation.md[Install the `odo` binary].
+. link:https://openshiftdo.org/installation/[Install the `odo` binary].
```

Comment:
we can simply points to `installation.adoc`

Reply:
@surajnarwade changed

Reply:
:+1: